src/libs/ComposerUtils/getDraftComment.js

```diff
@@ -0,0 +1,24 @@
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+
+const draftCommentMap = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT,
+    callback: (value, key) => {
+        if (!key) return;
+
+        const reportID = key.replace(ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT, '');
+        draftCommentMap[reportID] = value;
+    },
+});
+
+/**
+ * Returns a draft comment from the onyx collection.
+ * Note: You should use the HOCs/hooks to get onyx data, instead of using this directly.
+ * A valid use case to use this is if the value is only needed once for an initial value.
+ * @param {String} reportID
+ * @returns {String|undefined}
+ */
+export default function getDraftComment(reportID) {
```

Comment:
I tried to explain this in the comment above. The use case here is, that we only need the value for the initial value of a state variable. If we'd use the HOC the component would re-render every time the value (draft comment) changes, which is something we specifically want to avoid for performance reasons.

Reply:
The issue was discussed here: https://expensify.slack.com/archives/C01GTK53T8Q/p1692620941376059